src/test/java/com/puppycrawl/tools/checkstyle/api/FilterSetTest.java

```diff
@@ -117,6 +118,17 @@ public void testUnmodifiableSet() {
             () -> filterSet.getFilters().add(filter));
     }
 
+    /*
+      Due to low level configuration setup of FilterSet, conventional
+      input validation cannot be done here hence, pure JUnit testing has been
+      done for the time being
+    */
```

Comment:
@nrmancuso 

How can the message be improved here?
The same discussion is underway in https://github.com/checkstyle/checkstyle/pull/12884#discussion_r1145705481

Reply:
?? Sorry. I didn't understand. Am I supposed to use that message in the multiline comment.

Reply:
yes